### ISSUE TITLE
Make AEX-2 implementation somehow warn if used different networks

### ISSUE THREAD

**Issue by intnick** [aeternity/aepp-sdk-js#1068]
**Describe the bug**
Calling a getBalance method from my react app (via AEX2) gets a wrong balance. I have around 240Ae, and the method returns that I have 5Ae.
Using Superhero wallet, latest version, on the latest Chrome on windows 10.

**To Reproduce**
1. Create an app similar to the AEX2 example in Vue
2. Call the getBalance method on the client
3. Verify that the given balance is not correct

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/1571030/90886769-569cf700-e3b3-11ea-8b15-27f9bd5e9da5.png)

**Please tell us about your environment:**
  - superhero wallet v0.3.2
  - Compiler version: latest
  - SDK Version: v7.6.0

**Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. forum, telegram, etc)

**Comment by davidyuk** [comment]
It looks like you are connected to different networks in the wallet and in your app:

```bash
$ curl https://testnet.aeternity.io/v2/accounts/ak_2TKYxRRUdwCH2jFX9go6Se4GVbKUFvyGn4r6mVS1CbrQJ6DWds
{
  "balance":5000000000000000000,
  "id":"ak_2TKYxRRUdwCH2jFX9go6Se4GVbKUFvyGn4r6mVS1CbrQJ6DWds",
  "kind":"basic",
  "nonce":0,
  "payable":true
}
$ curl https://mainnet.aeternity.io/v2/accounts/ak_2TKYxRRUdwCH2jFX9go6Se4GVbKUFvyGn4r6mVS1CbrQJ6DWds
{
  "balance":244267945084000000000,
  "id":"ak_2TKYxRRUdwCH2jFX9go6Se4GVbKUFvyGn4r6mVS1CbrQJ6DWds",
  "kind":"basic",
  "nonce":12,
  "payable":true
}
```

```js
> new BigNumber('5000000000000000000').shiftedBy(-18).toString()
"5"
> new BigNumber('244267945084000000000').shiftedBy(-18).toString()
"244.267945084"
```

So, I think the problem is on the side of your aepp. Though AEX-2 should prevent a such case, to investigate that would be nice to see how exactly you are using sdk.

**Comment by intnick** [comment]
Yes, I removed the testnet from the configuration and now it works ok. Thanks for tracing the issue.
I used the code from the example, where it is also specifying both testnet and mainnet so I didn't expect this type of a bug, especially since superhero wallet doesn't have the testnet network connected.
